tests/golden/openshift4/crossplane/crossplane/01_helmchart/crossplane/templates/deployment.yaml

```diff
@@ -54,6 +71,31 @@ spec:
         volumeMounts:
         - mountPath: /cache
           name: package-cache
+      initContainers:
+      - args:
+        - core
+        - init
+        env:
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        image: docker.io/crossplane/crossplane:v1.7.1
+        imagePullPolicy: IfNotPresent
+        name: crossplane-init
+        resources:
+          limits:
+            cpu: 1000m
+            memory: 512Mi
+          requests:
+            cpu: 500m
+            memory: 256Mi
+        securityContext:
+          allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: true
+          runAsGroup: 65532
+          runAsUser: 65532
```

Comment:
Looks like the OCP4 post-processing filter needs to be updated to also remove `runAsUser` from init containers.